Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pwbox"
-version = "0.1.4"
+version = "0.1.5"
```

Comment:
Since `rand` / `rand_core` types are exposed in the public interface of the crate, bumping their version is a breaking change. Thus, the version should be `0.2.0`.